Documentation: Inbox

pawohl opened the pull request:
Document the **InboxScannerService**. Otherwise it may go entirely unnoticed by opencast administrators.

pawohl committed:
Documentation: Inbox

lkiesow commented on docs/guides/admin/docs/configuration/inbox.md:
```diff
@@ -0,0 +1,121 @@
+-----------------------------------------------
+
+Adjust
+[etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg](`https://github.com/opencast/opencast/blob/develop/etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg`).
```
You probably do not want to use ticks around the link.

pawohl replied:
```suggestion
[etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg](https://github.com/opencast/opencast/blob/develop/etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg).
```

lkiesow commented on docs/guides/admin/docs/configuration/inbox.md:
```diff
@@ -0,0 +1,121 @@
+-----------------------------------------
+
+In order to test the inbox scanner service, either put valid media package
+zip into the scanned directory or a single media file.
```
```suggestion
zip or a single media file into the scanned directory.
```

lkiesow commented on docs/guides/admin/docs/configuration/inbox.md:
```diff
@@ -0,0 +1,121 @@
+
+Note that even if the poll interval is small, it may take a little longer until
+the media package is added to the epsisode admin UI view because extract and
+copy operations may take some time.
```
“episode” and a suggestion for the sentence:

> …the media package is visible in the admin interface because extracting and/or
>￼ copying the media files will take some time.

pawohl replied:
```suggestion
copying the media files will take some time.
```

lkiesow commented on docs/guides/admin/docs/configuration/inbox.md:
````diff
@@ -0,0 +1,121 @@
+Create the follwing files:
+
+_manifest.xml_
+```
````
Maybe use:

```
```xml 
…
```

…and indent the xml?

pawohl replied:
```suggestion
```xml
```

lkiesow commented on docs/guides/admin/docs/configuration/inbox.md:
````diff
@@ -0,0 +1,121 @@
+```
+<?xml version="1.0" encoding="utf-8"?>
+<mediapackage xmlns:oc="http://mediapackage.opencastporject.org">
+<title>A media package courtesy by the inbox scanner.</title>
````
This title should probably match the dublin core title.

pawohl replied:
Changing dublin core title.

lkiesow commented on docs/guides/admin/docs/configuration/inbox.md:
````diff
@@ -0,0 +1,121 @@
+</mediapackage>
+```
+
+Note: You may create a valid empty media package using the
````
```suggestion
Note: You can create a valid empty media package using the
```

lkiesow commented on docs/guides/admin/docs/configuration/inbox.md:
```diff
@@ -0,0 +1,121 @@
+
+Besides ingesting media packages using the REST service of the IngestService,
+dedicated inbox directories located in the file system can be scanned by
+Opencast. This, for example allows adding media packages to Opencast by copying
```
```suggestion
Opencast. This, for example, allows adding media packages to Opencast by copying
```

lkiesow reviewed (changes requested):
Thanks for the patch. It looks good except for a few minor typos, …

pawohl commented on docs/guides/admin/docs/configuration/inbox.md:
```diff
@@ -0,0 +1,121 @@
+zip into the scanned directory or a single media file.
+
+Note that even if the poll interval is small, it may take a little longer until
+the media package is added to the epsisode admin UI view because extract and
```
```suggestion
the media package is visible in the admin interface because extracting and/or
```

pawohl commented on docs/guides/admin/docs/configuration/inbox.md:
```diff
@@ -0,0 +1,121 @@
+   xmlns:oc="http://www.opencastproject.org/matterhorn"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance/"
+   xsi:schemaLocation="http://www.opencastproject.org http://www.opencastproject.org/schema.xsd">
+<dcterms:title>Some Episode title</dcterms:title>
```
```suggestion
<dcterms:title>A media package courtesy by the inbox scanner.</dcterms:title>
```

lkiesow committed:
Documentation: Inbox: Code review

Fix issues detected during code review, including
language shortcomings, formatting issues and typos.

pawohl commented:
@lkiesow Thanks for the review. The "requested changes" feature by GitHub is nice, though I cannot automatically apply them.

I noticed there is no page dedicated to the REST API in the docs (neither admin nor developer).
